### PR TITLE
add library models

### PR DESCRIPTION
**Pull request by manjotchahal**

### CODE REVIEW

File: src/Housing.Forecast.Library/Housing.Forecast.Library.csproj [hunk 1]
```diff
@@ -5,4 +5,17 @@
...
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.0.3" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.0.3" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.0.3" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="2.0.3" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="System.ComponentModel.Annotations">
+      <HintPath>..\..\..\..\..\..\..\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.netcore.app\2.0.0\ref\netcoreapp2.0\System.ComponentModel.Annotations.dll</HintPath>
```
[the remark this answers is not in the document]
Comment (escalonn) [reply src/Housing.Forecast.Library/Housing.Forecast.Library.csproj]: however, should not even need dataannotations in library project, i don't think.

[the remark this answers is not in the document]
Comment (manjotchahal) [reply src/Housing.Forecast.Library/Housing.Forecast.Library.csproj]: We fixed the issue but we do need to add data annotations to our model classes in the library project, correct?

[the remark this answers is not in the document]
Comment (escalonn) [reply src/Housing.Forecast.Library/Housing.Forecast.Library.csproj]: the ones relating to db access, definitely not. the more abstract ones - only if you are actually using them for validation within the library project, which you aren't.

[the remark this answers is not in the document]
Comment (manjotchahal) [reply src/Housing.Forecast.Library/Housing.Forecast.Library.csproj]: So we should have separate data models within our context project, correct? To make use of those db access annotations.

[the remark this answers is not in the document]
Comment (escalonn) [reply src/Housing.Forecast.Library/Housing.Forecast.Library.csproj]: yes

**Pull request merged**